share/rocm/cmake/ROCMCreatePackage.cmake

```diff
@@ -10,6 +10,37 @@ include(GNUInstallDirs)
 find_program(MAKE_NSIS_EXE makensis)
 find_program(RPMBUILD_EXE rpmbuild)
 find_program(DPKG_EXE dpkg)
+find_program(GIT NAMES git)
+
+function(rocm_get_git_commit_tag OUTPUT_VERSION)
```

Comment:
This should be in the `rocm_setup_version` as the `PROJECT_VERSION_TWEAK` should be set to the commit tag.

Reply:
done. Please review the latest changes.